### ISSUE TITLE
React Native transform error

### ISSUE THREAD

**Issue by charlie-axsy** [join-monster/join-monster#332]
React Native 0.49 throws an error when importing join monster.  The reason is that the React Native packager is unable to handle dynamic imports.  There is one case of this in join monster - https://github.com/acarl005/join-monster/blob/master/src/stringifiers/dispatcher.js#L16

As a result join monster cannot be used with React Native straight from npm.  To get around this I am applying a `fixJoinMonster.patch` file on postinstall to remove the dynamic require statement.

**Comment by charlie-axsy** [comment]
Ah, just found https://github.com/acarl005/join-monster/issues/172 which describes the same issue

**Comment by charlie-axsy** [comment]
I get this error even while using the `dialectModule` option.

```
const options = {
  dialectModule: require('join-monster/dist/stringifiers/dialects/sqlite3');
}
```

**Comment by GlennMatthys** [comment]
Are your dependencies up to date?

**Comment by nicoabie** [comment]
This was fixed in version 2.0.4
